### PR TITLE
Add Margins Conference

### PR DESCRIPTION
**Pull request by erinosher**
Changing the home page to add @HackSocNotts/margins-conf

### CODE REVIEW

**Comment by erinosher** [comment]
@niceotherwise

File: src/components/pages/HomePage/HomePage.jsx [hunk 1]
```diff
@@ -96,11 +96,12 @@ const HomePage = () => (
           </Grid.Column>
           <Grid.Column className={styles.flagshipGrid}>
             <Header as='h3' className={styles.h3}>
-              inspireWiT
+              margins
```
Comment (DanielCordell) [inline src/components/pages/HomePage/HomePage.jsx:104]: Should these maybe have hyperlinks?

File: src/components/pages/HomePage/HomePage.jsx [hunk 2]
```diff
@@ -90,17 +90,18 @@ const HomePage = () => (
...
             </Header>
             <p className={styles.p}>
-              inspireWiT Conference is a technology-focused day for women and people of all
-              gender minorities—inspiring everyone!
+              margins, the successor to Women in Tech and inspireWiT,  is a free one-day conference
+              for people from marginalised backgrounds that work in/with/around technology (or hope
+              to break into the industry!). Learn more about it <a href='https://margins.tech' target='_blank' rel='noopener noreferrer'>here</a>.
```
Comment (erinosher) [inline src/components/pages/HomePage/HomePage.jsx]: How's this? @DanielCordell

File: src/components/pages/HomePage/HomePage.jsx [hunk 3]
```diff
@@ -90,17 +90,18 @@ const HomePage = () => (
               HackNotts
             </Header>
             <p className={styles.p}>
-              HackNotts is a 24 hour hackathon from Nov 24-25 at the University of Nottingham, in
-              England. Over 150 students will team up to learn about tech and create cool stuff.
+              HackNotts is a 24 hour hackathon at the University of Nottingham, in
+              England. Over 150 students will team up to learn about tech and create cool stuff. Learn more about it <a href='https://hacknotts.com' target='_blank' rel='noopener noreferrer'>here</a>.
```
Comment (erinosher) [inline src/components/pages/HomePage/HomePage.jsx]: Also did it for HackNotts

**Review by DanielCordell — APPROVED** [review]

**Pull request merged**